assets/css/structures/_definition-content.scss

```diff
@@ -31,17 +31,18 @@
 
     & > p {
       font-family: $sans-serif;
-      font-size: 1.5rem;
+      font-size: var(--step0);
```

Comment:
I think you changed this here because `1.5rem` made the first paragraph in definition mostly unreadable on mobile?